### DISCUSSION TITLE
Support `input-error-content` and et al.

### DISCUSSION THREAD

**Discussion by ghost** [saadeghi/daisyui#3123] [Ideas / feature requests]
I see that DaisyUI introduces a pretty cool feature as `input-primary`, `input-error`, etc. to build the color for the input. In the same manner, I expect it would work with `input-error-content` but it does not.

Do you think it's helpful to support `input-error-content`? In my POV, it will achieve a consistent DX to DaisyUI.

**Comment by saadeghi** [comment]
`error-content` is the foreground color which is supposed to be used if the background is `error` color (to have an accessible contrast).  
However `input-error` is not using the red color as the background. It's the border.

Can you tell me about the design you want to build? How do you want the input to look like?

**Comment by ghost** [comment]
Here is a reasonable use-case I think.

<img width="379" alt="Screenshot 2024-07-04 at 14 57 39" src="https://github.com/saadeghi/daisyui/assets/71456318/dda3313f-1b25-4961-a717-3ed587dc0b6a">

p/s: img updated
